openstack/cerebro/cerebro-main/templates/dc-state-service.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: dc-state-service
+  labels:
+    app: dc-state-service
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "{{ .Values.dcStateService.prometheusClientPort }}"
+spec:
+  type: NodePort
+  ports:
+    - targetPort: 80
+      port: 80
+  selector:
+    app: dc-state-service
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dc-state-service
+  labels:
+    app: dc-state-service

```

Comment:
is this missing the `data` part?

Reply:
I'm not sure which `data` part you are referring to. The helm chart is able to run locally

Reply:
You are embedding this in your deployment with `envFrom`, but there is no data in here, where you can derive an environment variable. That's why I've asked for a `data` part.